index.html

```diff
@@ -2,160 +2,217 @@
+        </div>
+    </section>
+    <section class="about">
+        <div class="content">
+            <h2>About <br> Myself</h2>
+            <p class="description">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <h5>LET'S CONNECT</h5>
+            <ul class="links">
+                <li><a title="linkdin" href="https://www.linkedin.com/in/naji-ali-55b5421b3/"><i
+                            class="fa-brands fa-linkedin fa-2x"></i></a></li>
+                <li><a title="github" href="https://github.com/naji-97"><img src="./assest/icons/Vector.svg" alt=""
+                            srcset=""></a></li>
+                <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (3).svg"
+                            alt="" srcset=""></a></li>
+                <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (4).svg"
+                            alt="" srcset=""></a></li>
+                <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (2).svg"
+                            alt="" srcset=""></a></li>
+            </ul>
+            <div class="see-resume">
+                <button><a href="#">See resume</a></button>
+            </div>
+            <div class="languages">
+                <div class="flex">
+                    <p>Languages</p>
+                    <i class="fa-solid fa-angle-down "></i>
+                </div>
+                <ul class="mycompetence flex">
+                    <li class="flex"><img src="/assest/img/Ellipse 1.png" alt=""> JavaScript</li>
+                    <li class="flex"> <img src="/assest/img/Ellipse 1 (1).png" alt="">HTML</li>
+                    <li class="flex"> <img src="/assest/img/Ellipse 1 (2).png" alt="">CSS</li>
+                </ul>
+                <div class="flex border-bootom">
+                    <p>Framwrks</p>
+                    <i class="fa-solid fa-angle-down "></i>
+                </div>
+                <div class="flex border-bootom">
+                    <p>Skills</p>
+                    <i class="fa-solid fa-angle-down "></i>
+                </div>
+            </div>
+        </div>
+        </div>
+    </section>
+
+    <section class="contact-form">
+        <div class="contac-card">
+            <div class="contact-me">
+                <h2>contact me</h2>
+
+                <p>
+                    if you have an application you are interested in developing, a
+                    features that you need built or a project that need coding. I'd love
+                    to help with it.
+                </p>
+            </div>
+            <div class="contact-me-form">
+                <form>
```

Comment:
- [ ] In your current form condition, when I submit the form nothing happens 🤔. Please make sure that you use [Formspree Service](https://formspree.io/html) to collect data, and also to integrate your from with this service use the `action` and `method` attributes 📝.

![image](https://user-images.githubusercontent.com/43172164/207341704-a10c8406-c3ee-4934-8183-8304aa2d339c.png)